src/image-helpers.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @class ImageHelpers
+ */
+export default class ImageHelpers {
+
+  /**
+   * Generates the image src for a resized image with maximum dimensions `maxWidth` and `maxHeight`.
+   * Images do not scale up.
+   *
+   * @method imageForSize
+   * @param {Object} image the original image model to generate the image src for
+   * @param {Integer} maxHeight the maximum height for the image
+   * @param {Integer} maxWidth the maximum width for the image
+   * @return {String} The image src for the resized image
+   */
+  imageForSize(image, maxHeight, maxWidth) {
+    const urlTokens = image.src.split('/');
+    const imageTokens = urlTokens[urlTokens.length - 1].split('.');
+
+    // Take the first token and append the dimensions
+    imageTokens[0] = imageTokens[0].concat(`_${maxHeight}x${maxWidth}`);
```

Comment:
Does this assumption hold if there's multiple dots?

`cat.foo.bar.jpg`. Should it still append the dimensions on the first dot?